src/Uno.UWP/UI/Core/CoreWindow.macOS.cs

```diff
@@ -13,7 +13,8 @@ namespace Windows.UI.Core
 {
 	public partial class CoreWindow 
 	{
-        private readonly NSWindow _window;
+		// HACK: TODO: Find a better way to get this reference
+		public readonly NSWindow _window;
```

Comment:
I'm not sure what to do about this exactly. It should at least be made 'internal'; however, I need a global way to access the reference to the NSWindow.

Reply:
Should definitely be `internal` (and converted to a property like `NativeWindow`). 

Otherwise it's 100% legit to need access to the native window (in internal code), and you can either keep it here, either move it to the `Window.UI.Xaml.Window`. Personally I think it's better here, but it might be easier to discover in the other one.

Reply:
Ok, if there aren't any concerns adding a new internal property, I'll go ahead and add it!